src/lib/link.ts

```diff
@@ -406,6 +419,7 @@ export class Link {
 
   // Returns the last height that this side knows of the other blockchain
   public async lastKnownHeader(side: Side): Promise<number> {
+    this.logger.info(`Get last known header for side ${side}`);
```

Comment:
This should be verbose.
Actually, I think most of the queries (besides the updateClient ones) should be on verbose level